docs/modules/ROOT/pages/operations/result-frames.adoc

```diff
@@ -26,7 +26,7 @@ To see the results, switch the mode to the *Table* view.
 == Graph result frame
 
 The graph visualization functionality is designed to display a node-graph representation of the underlying data stored in the database in response to a given Cypher query.
-It shows circles for nodes and lines for relationships, and is especially useful for determining areas of interest or quickly assessing the current state and organization of the data.
+It shows circles for nodes and lines for relationships and is especially useful for determining areas of interest or quickly assessing the current state and organization of the data.
```

Comment:
```suggestion
It shows circles for nodes and lines for relationships and is especially useful for determining areas of interest or quickly assessing the current state and structure of the data.
```

Reply:
I haven't read it, just fixed the auto-suggested comma 😄, but I'll read it now. Thanks, @martin-neotech!